infrastructure/prime-app-ephemeral-template.yml

```diff
@@ -755,7 +754,10 @@ objects:
               - createdb -O $(APPUSER) prime-${SVC_NAME}
             env:
               - name: PGHOST
-                value: dev-patroni
+                valueFrom:
+                  configMapKeyRef:
+                    name: database-cm
+                    key: database-host
               - name: PGDATABASE
                 value: postgres
```

Comment:
Rather than `postgres`, should this be `prime-${SVC_NAME}`?

Reply:
This is for creating the prime-${SVC_NAME} with init-container. We are logging in to PostgreSQL as superuser, into default postgres database. prime-${SVC_NAME} does not exist yet.

Reply:
Can you please add a comment re: that?

Reply:
See above for my comment